torchtune/modules/tokenizers/_utils.py

```diff
@@ -42,6 +42,12 @@ def truncate(
     max_seq_len: int,
     eos_id: Union[int, bool],
 ):
+    """
```

Comment:
actually, do you mind removing this function? this is not used anywhere afaik and is duplicated in `torchtune.data.truncate`. I thought I had removed this but for some reason it remains...